app/org/sagebionetworks/bridge/services/SharedModuleService.java

```diff
@@ -120,7 +120,7 @@ private SharedModuleImportStatus importModule(StudyIdentifier studyId, SharedMod
             long sharedSurveyCreatedOn = metadata.getSurveyCreatedOn();
             GuidCreatedOnVersionHolder sharedSurveyKey = new GuidCreatedOnVersionHolderImpl(sharedSurveyGuid,
                     sharedSurveyCreatedOn);
-            Survey sharedSurvey = surveyService.getSurvey(sharedSurveyKey, true);
+            Survey sharedSurvey = surveyService.getSurvey(studyId, sharedSurveyKey, true, true);
```

Comment:
Similarly, based on the schema branch above, this code is supposed to get the survey from the shared study and then create it in the local study. So this should be BridgeConstants.SHARED_STUDY_ID.

Reply:
Now using the constant for the shared study